classes/invitation/invitations/userRoleAssignment/resources/BaseUserRoleAssignmentInviteResource.php

```diff
@@ -91,4 +94,36 @@ protected function createNewUserFromPayload(UserRoleAssignmentInvitePayload $pay
 
         return $newUser;
     }
+
+    protected function transformCurrentUserGroups(int $id , Context $context): array
+    {
+        $userGroups = [];
+        $userUserGroups = UserUserGroup::query()
+            ->withUserId($id)
+            ->withContextId($context->getId())
+            ->get()
+            ->toArray();
+        foreach ($userUserGroups as $key => $userUserGroup) {
+            $userGroup = Repo::userGroup()
+                ->get($userUserGroup['userGroupId'])
+                ->toArray();
+            $userGroups[$key] = $userUserGroup;
+            $userGroups[$key]['masthead'] = $userUserGroup['masthead'] === 1;
+            $userGroups[$key]['name'] = $userGroup['name'][Locale::getLocale()];
+            $userGroups[$key]['id'] = $userGroup['userGroupId'];
+        }
+        return $userGroups;
+    }
+
+    /**
+     * get user working languages
+     * @param Context $context
+     * @param $userLocales
+     * @return string
+     */
+    protected function transformWorkingLanguages(Context $context,$userLocales): string
```

Comment:
Here can we define all the parameters types? 

Also could you leave an empty space after the `,` here and elsewhere?